tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -271,233 +271,233 @@
+    <string name="pref_tor_exit_nodes" translatable="false">Exit Nodes</string>
+    <string name="pref_tor_exit_nodes_summ">A list of country codes to use as exit nodes (those delivering traffic for you outside the Tor network).</string>
+    <string name="pref_tor_exclude_nodes" translatable="false">Exclude Nodes</string>
     <string name="pref_tor_exclude_nodes_summ">A list of country codes to avoid when building a circuit.</string>
-    <string name="pref_tor_entry_nodes" translatable="false">EntryNodes</string>
+    <string name="pref_tor_entry_nodes" translatable="false">Entry Nodes</string>
     <string name="pref_tor_entry_nodes_summ">A list of country codes of nodes to use for the first hop in your normal circuits.</string>
-    <string name="pref_tor_entry_nodes_alt_summ">Disable Tor bridges if you want to select entry nodes.</string>
-    <string name="pref_tor_strict_nodes" translatable="false">StrictNodes</string>
-    <string name="pref_tor_strict_nodes_summ">If StrictNodes enabled, Tor will treat solely the ExcludeNodes option as a requirement to follow for all the circuits you generate, even if doing so will break functionality for you.</string>
-    <string name="pref_tor_fascist_firewall" translatable="false">FascistFirewall</string>
-    <string name="pref_tor_fascist_firewall_summ">If enabled, Tor will only create outgoing connections to ORs running on ports that your firewall allows (defaults to 80 and 443).</string>
-    <string name="pref_tor_new_circuit_period" translatable="false">NewCircuitPeriod</string>
-    <string name="pref_tor_new_circuit_period_summ">Every NUM seconds consider whether to build a new circuit.</string>
-    <string name="pref_tor_max_circuit_dirtiness" translatable="false">MaxCircuitDirtiness</string>
-    <string name="pref_tor_max_circuit_dirtiness_summ">Feel free to reuse a circuit that was first used at most NUM seconds ago, but never attach a new stream to a circuit that is too old.</string>
-    <string name="pref_tor_enforce_distinct_subnets" translatable="false">EnforceDistinctSubnets</string>
-    <string name="pref_tor_enforce_distinct_subnets_summ">If enabled, Tor will not put two servers whose IP addresses are "too close" on the same circuit. Currently, two addresses are "too close" if they lie in the same /16 range.</string>
-    <string name="pref_tor_enable_SOCKS_proxy">Enable SOCKS proxy</string>
-    <string name="pref_tor_SOCKS_port" translatable="false">SOCKSPort</string>
-    <string name="pref_tor_SOCKS_port_summ">Open this port to listen for connections from SOCKS-speaking applications.</string>
-    <string name="pref_tor_enable_HTTPTunnel_proxy">Enable HTTPTunnel</string>
-    <string name="pref_tor_HTTPTunnel_port" translatable="false">HTTPTunnelPort</string>
+    <string name="pref_tor_entry_nodes_alt_summ">Turn off \"Tor bridges\" if you want to select entry nodes yourself.</string>
```

Comment:
```suggestion
    <string name="pref_tor_entry_nodes_alt_summ">Turn off all Tor bridges if you want to select entry nodes yourself.</string>
```
On the assumption that this is not a setting. There are other instances of similar confusion on my part.